deepchem/molnet/load_function/qm8_datasets.py

```diff
@@ -8,7 +8,7 @@
 from typing import List, Optional, Tuple, Union
 
 GDB8_URL = "https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/gdb8.tar.gz"
-QM8_CSV_URL = "https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/qm8.csv"
+QM8_CSV_URL = "https://gist.githubusercontent.com/ellagale/9e632c9fcf11aaa803ae8a86f9c2a82d/raw/dd862e24858fe68141021c843069ecd0350b5d13/qm8.csv"

```

Comment:
Could we change this to point to the actual quantum machine website? I want to make sure we point to a stable data source for the CSV